lib/osf-components/addon/components/node-card/template.hbs

```diff
@@ -40,7 +40,9 @@
                 {{else if (eq @node.revisionState 'in_progress')}}
                     <span class='label label-danger'>{{t 'node_card.registration.statuses.revision_in_progress'}}</span> |
                 {{else if (eq @node.revisionState 'pending_moderation')}}
-                    <span class='label label-danger'>{{t 'node_card.registration.statuses.revision_pending_moderation'}}</span> |
+                    {{#if (not-eq @node.reviewsState 'pending_moderation')}}
```

Comment:
I feel like this could be more concise somehow... _unless_ this was done intentionally 🌚

Reply:
I believe `if (not-eq)` is better than `unless (eq)`. Also, using another helper with `unless` is a linting err.